File: Selenium/Activities/Activity9.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/keyboard-events")
    print(f"the title of the main page is: {driver.title}")

    actions= webdriver.ActionChains(driver)
    actions.send_keys("This is a selenium script").perform()

    message = driver.find_element(By.CSS_SELECTOR,"h1.mt-3")
    print(f"the message on the page is: {message.text}")

    driver.quit()

File: Selenium/Activities/Activity9.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity9 {
	public static void main(String args[]) {
		WebDriver driver = new FirefoxDriver();
		driver.get("https://training-support.net/webelements/keyboard-events");
		System.out.println("The title of the main page: "+driver.getTitle());
		
		Actions builder = new Actions(driver);
		builder.sendKeys("This is a selenium script").build().perform();
		
		
		WebElement message = driver.findElement(By.cssSelector("h1.mt-3"));
		
		System.out.println("The text is: "+message.getText());
		
		driver.quit();
		
	}

}
